examples/CMakeLists.txt

```diff
@@ -81,10 +81,11 @@ set(DATA_SHADERS
     ex_prim_shader_pixel.hlsl
     ex_prim_shader_vertex.glsl
     ex_prim_shader_vertex.hlsl
-    ex_shader_multitex_pixel.glsl
-    ex_shader_multitex_pixel.hlsl
     ex_prim_wrap_pixel.glsl
     ex_prim_wrap_pixel.hlsl
+    ex_shader_multitex_pixel.glsl
+    ex_shader_multitex_pixel.hlsl
+    ex_shader_palette_pixel.glsl
     ex_shader_pixel.glsl
```

Comment:
I had a really old branch, so when I rebased I had a conflict here (it was simple). Anyway, before the rebase I saw these files were alphabetical (in my older checkout), and so after the rebase I again sorted alphabetically in my IDE. Looking at the diff now, obv it wasn't alpha sorted in master... anyway, I can revert the order changes if requested.